lib/matplotlib/layout_engine.py

```diff
@@ -203,7 +203,7 @@ def __init__(self, *, h_pad=None, w_pad=None,
         Parameters
         ----------
         h_pad, w_pad : float
-            Padding around the axes elements in figure-normalized units.
+            Padding around the axes elements in inches.
```

Comment:
We did this one at #24981

Reply:
I wasn't being that subtle - I didn't catch that it was done in #24981.